src/components/NavBar/SearchBar.css.ts

```diff
@@ -23,6 +26,35 @@ const baseSearchStyle = style([
   },
 ])
 
+const baseSearchNftStyle = style([
+  sprinkles({
+    paddingY: '12',
+    width: { sm: 'viewWidth' },
+    borderStyle: 'solid',
+    borderWidth: '2px',
+    borderColor: 'backgroundOutline',
+  }),
+  {
+    '@media': {
+      [`screen and (min-width: ${breakpoints.sm}px)`]: {
+        width: `${DESKTOP_NAVBAR_WIDTH}px`,
+      },
```

Comment:
These styles are just duplicates of `baseSearchStyle` so you can replace this section with importing that style

Reply:
Alternatively you could delete that section and then in `nftSearchBar` just import `baseSearchStyle` and these added breakpoints